index.js

```diff
@@ -1,106 +1,100 @@
 'use strict'
 
 const eu = encodeURIComponent
-const figgyPudding = require('figgy-pudding')
-const getStream = require('get-stream')
 const npmFetch = require('npm-registry-fetch')
 const validate = require('aproba')
 
-const TeamConfig = figgyPudding({
-  description: {},
-  Promise: { default: () => Promise }
-})
-
 const cmd = module.exports = {}
 
-cmd.create = (entity, opts) => {
-  opts = TeamConfig(opts)
-  return pwrap(opts, () => {
+cmd.create = (entity, opts = {}) => {
+  return Promise.resolve().then(() => {
     const { scope, team } = splitEntity(entity)
     validate('SSO', [scope, team, opts])
-    return npmFetch.json(`/-/org/${eu(scope)}/team`, opts.concat({
+    const uri = `/-/org/${eu(scope)}/team`
+    return npmFetch.json(uri, {
+      ...opts,
       method: 'PUT',
       scope,
       body: { name: team, description: opts.description }
-    }))
+    })
   })
 }
 
-cmd.destroy = (entity, opts) => {
-  opts = TeamConfig(opts)
-  return pwrap(opts, () => {
+cmd.destroy = (entity, opts = {}) => {
+  return Promise.resolve().then(() => {
     const { scope, team } = splitEntity(entity)
     validate('SSO', [scope, team, opts])
-    return npmFetch.json(`/-/team/${eu(scope)}/${eu(team)}`, opts.concat({
+    const uri = `/-/team/${eu(scope)}/${eu(team)}`
+    return npmFetch.json(uri, {
+      ...opts,
       method: 'DELETE',
       scope
-    }))
+    })
   })
 }
 
-cmd.add = (user, entity, opts) => {
-  opts = TeamConfig(opts)
-  return pwrap(opts, () => {
+cmd.add = (user, entity, opts = {}) => {
+  return Promise.resolve().then(() => {
     const { scope, team } = splitEntity(entity)
     validate('SSO', [scope, team, opts])
-    return npmFetch.json(`/-/team/${eu(scope)}/${eu(team)}/user`, opts.concat({
+    const uri = `/-/team/${eu(scope)}/${eu(team)}/user`
+    return npmFetch.json(uri, {
+      ...opts,
       method: 'PUT',
       scope,
       body: { user }
-    }))
+    })
   })
 }
 
-cmd.rm = (user, entity, opts) => {
-  opts = TeamConfig(opts)
-  return pwrap(opts, () => {
+cmd.rm = (user, entity, opts = {}) => {
+  return Promise.resolve().then(() => {
     const { scope, team } = splitEntity(entity)
     validate('SSO', [scope, team, opts])
-    return npmFetch.json(`/-/team/${eu(scope)}/${eu(team)}/user`, opts.concat({
+    const uri = `/-/team/${eu(scope)}/${eu(team)}/user`
+    return npmFetch.json(uri, {
+      ...opts,
       method: 'DELETE',
       scope,
       body: { user }
-    }))
+    })
   })
 }
 
-cmd.lsTeams = (scope, opts) => {
-  opts = TeamConfig(opts)
-  return pwrap(opts, () => getStream.array(cmd.lsTeams.stream(scope, opts)))
+cmd.lsTeams = (scope, opts = {}) => {
+  return Promise.resolve().then(() => {
```

Comment:
Ok, scratch what I said above, this one is definitely silly.  Could just be a one-liner:

```js
cmd.lsTeams = (...args) => cmd.lsTeams.stream(...args).collect()
```